relay_rpc/Cargo.toml

```diff
@@ -26,32 +26,32 @@ derive_more = { version = "0.99", default-features = false, features = [
     "as_ref",
     "as_mut",
 ] }
-serde = { version = "1.0", features = ["derive", "rc"] }
-serde-aux = { version = "4.1", default-features = false }
-serde_json = "1.0"
-thiserror = "1.0"
+serde = { version = "1", features = ["derive", "rc"] }
+serde-aux = { version = "4", default-features = false }
+serde_json = "1"
+thiserror = "1"
 ed25519-dalek = { git = "https://github.com/dalek-cryptography/ed25519-dalek.git", rev = "7529d65" }
 rand = "0.7"
 chrono = { version = "0.4", default-features = false, features = [
     "std",
     "clock",
 ] }
-regex = "1.7"
-once_cell = "1.16"
-jsonwebtoken = "8.1"
+regex = "1"
```

Comment:
So what happens if `relay_rpc` is using features added in `1.7` release, while the consumer code forces e.g. `1.0` version?

Reply:
If we needed features/fixes from the 1.7 release, then we would set the minimum to 1.7 not 1 as I did